app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :recoverable
```

Comment:
it's probably worth mentioning in the commit message that you are including ([mixing in the behaviours from](https://ruby-doc.org/docs/ruby-doc-bundle/ProgrammingRuby/book/tut_modules.html#S2)) these two modules: 'database_authenticatable' and 'recoverable' -- and explaining briefly what they each are?